### ISSUE TITLE
Add support for https endpoints

### ISSUE THREAD

**Issue by davissp14** [kragniz/python-etcd3#340]
https://github.com/kragniz/python-etcd3/blob/master/etcd3/client.py#L108

As of right now if no certs are provided it's assumed it's an insecure connection.  We should be setting the default grpc channel credentials in the event https is present without certs.

https://github.com/davissp14/etcdv3-ruby/blob/master/lib/etcdv3/connection.rb#L41-L47

**Comment by davissp14** [comment]
Looks like this may be what we want here: 
https://grpc.io/grpc/python/grpc.html#grpc.ssl_channel_credentials

`The PEM-encoded root certificates as a byte string, or None to retrieve them from a default location chosen by gRPC runtime.`

**Comment by mattzque** [comment]
seems like the library is not actively developed anymore?
I've got the same issue, simple fix would be something like this: https://github.com/mattzque/python-etcd3/commit/36d5e3504b012b7e1794e34e1b460f39c5e8d0ca
and then passing ca_cert=True, that will then specify None to ssl_channel_credentials

Multiple hostnames would be another critical feature, but that also seems to have stalled https://github.com/kragniz/python-etcd3/pull/106

**Comment by Raphael-C-Almeida** [comment]
@mattzque I agree with you. I really need #106 and it seems stalled. Is there anyone who can inform if the lib is actively developed or not? If it's active, an ETA for #106 changes would be nice. I'm using this lib for something critical, and if it's not being actively developed I have to choose another one.

**Comment by kragniz** [comment]
@Raphael-C-Almeida would you like to submit a PR for the changes you require?

**Comment by kragniz** [comment]
I think #106 is pretty stalled, it'd be great if you maybe could pick it up and try to get it into a mergable state.

I'm personally not using this library in production at the moment, so it's only receiving basic maintenance from me. New features are welcome, though, and I'm open to adding new maintainers.
